aten/src/ATen/native/quantized/cudnn/Linear.cpp

```diff
@@ -273,6 +308,9 @@ at::Tensor PackedLinearWeightCudnn::apply_impl(
     const at::Tensor& act,
     double output_scale,
     int64_t output_zero_point) {
+  TORCH_CHECK(act.dim() <= max_num_input_dim, "Number of dimensions for input tensor exeeds ", max_num_input_dim,
+  "; Increase max_num_input_dim in aten/src/ATen/native/quantized/cudnn/Linear.cpp to a larger value, if necessary, and recompile.");
```

Comment:
probably shouldn't say this, since TORCH_CHECK errors are the user errors, it happens when user didn't provide proper arguments, I think we can just remove this line. we should be able to figure out the constraint and increase this number

Reply:
removed